CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [v4.13.0](https://github.com/algolia/algoliasearch-client-javascript/compare/4.12.2...4.13.0)
+
+- fix(recommend): declare type before usage (#1399) ([a91cc5b](https://github.com/algolia/algoliasearch-client-javascript/commit/a91cc5b)), closes [#1399](https://github.com/algolia/algoliasearch-client-javascript/issues/1399)
```

Comment:
```suggestion
```

Reply:
Ah yes indeed it's not necessary to have it here, thanks